pkg/controller/metrics/deorbit.go

```diff
@@ -0,0 +1,36 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var DeorbitOperationsLatency = prometheus.NewSummaryVec(
```

Comment:
I think these need to be registered in an `init` function: See #195

Reply:
Yes, thanks.